src/components/navBar/index.tsx

```diff
@@ -131,8 +140,9 @@ const NavBar: React.FC<NavBarProps> = ({ tokens }) => {
   const LoggedInExtra = () => {
     return (
       <FlexDiv>
-        {/* This needs to changed, not a constant */}
-        <Paragraph style={{ margin: 'auto 20px auto 0' }}>Jack Blanc</Paragraph>
+        <Paragraph style={{ margin: 'auto 20px auto 0' }}>
```

Comment:
⛏️ Its a little odd that this is isnt a styled component but since its just margins I dont think it really matters